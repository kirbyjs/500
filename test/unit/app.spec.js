import React from 'react';
import { shallow } from 'enzyme';
import App from '../../src/app';


describe('<App /> rendering', () => {
    let component;

    beforeEach(() => {
        component = shallow(<App />);
    });

    it('App renders', () => {
        expect(component).toMatchSnapshot();
    });
});
