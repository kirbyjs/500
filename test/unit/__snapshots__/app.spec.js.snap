// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<header>
  <h1>
    500!
  </h1>
</header>
`;
